annotation_full.wdl

```diff
@@ -1,25 +1,26 @@
+version 1.0
+
 import "./structural-annotation.wdl" as sa
 import "./functional-annotation.wdl" as fa
 
 workflow annotation {
+input {
   String  proj
   String  input_file
   String  imgap_project_id
   String  database_location="/refdata/img/"
   String  imgap_project_type="metagenome"
-  String?  gm_license="/refdata/licenses/.gmhmmp2_key"
+  String  gm_license="/refdata/licenses/.gmhmmp2_key"
```

Comment:
This file should be left as optional, a user can also have the key in their homedir.

Reply:
If they have a key in their homedir, they can redefine the variable through json inputs. What are the advantages of having it as an optional variable vs pre-defined?